dockstore-webservice/src/main/java/io/dockstore/webservice/Helper.java

```diff
@@ -228,6 +236,102 @@ public String getContent() {
         return map;
     }
 
+    private static Container parseBitbucketCWL(HttpClient client, Container container, TokenDAO tokenDAO, User user) {
```

Comment:
Remember to comment your methods. Why is there a method for parsing CWL files specifically from bitbucket for example?